packages/ui/src/components/va-radio/types.ts

```diff
@@ -0,0 +1,4 @@
+export type option = {
```

Comment:
```suggestion
export type VaRadioOption = {
```

We can export this type to the end user.

```ts
const options: VaRadioOptions[] = []
```